Create releases.md

hesterbruikman opened the pull request:
### Summary
File that captures release notes for App store ad Play store. Informs users of relevant additions, changes, fixes.

hesterbruikman committed:
Create releases.md

File that captures release notes for App store ad Play store. Informs users of relevant additions, changes, fixes.

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 5066517d | [#1](https://ci.status.im/job/status-react/job/prs/job/android/job/PR-11463/1/) | 2020-11-23 11:31:17 | ~11 min | `android` | [:package:`apk`](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-201123-111939-506651-pr-universal.apk) [:calling:](https://chart.apis.google.com/chart?cht=qr&chs=400x400&chld=L%7C%0A1&chl=https%3A%2F%2Fstatus-im-prs.ams3.digitaloceanspaces.com%2FStatusIm-201123-111939-506651-pr-universal.apk)|

cammellos reviewed (commented):
looks ok, the only thing is that we tend to use capital letters for `md` files top level,so woudl be 
`RELEASES.md`

jakubgs commented on releases.md:
```diff
@@ -0,0 +1,889 @@
+# Mobile Release Notes
+
+###### tags: `mobile` `core`
```
This is probably just noise.

hesterbruikman replied:
thanks removing

jakubgs commented on releases.md:
```diff
@@ -0,0 +1,889 @@
+- [ ] Translations for comms available
+- [ ] Promotional content ready
+- [ ] Update Draft with release on https://github.com/status-im/status-react/releases
+- [ ] Add release to DO store and [update site](https://notes.status.im/s/status-release-upload#) (Jakub - requires credentials to upload) 
```
This checklist shouldn't be here. I'd move it to a separate file. Like `doc/RELEASE_CHECKLIST.md` or something.

hesterbruikman replied:
agreed, will move to a separate file

hesterbruikman committed:
Update and rename releases.md to RELEASES.md

As suggested, removing checklist to keep in a separate file and using capitals for the title
